index.html

```diff
@@ -0,0 +1,139 @@
+    <link rel="icon" href="images/music.png">
+    <title>E-Ticket</title>
+</head>
+<body>
+
+  <header id="header">
+      <div>
+        <ul class="top design">
+            <li class="icons"><i class="fa-brands fa-facebook-f"></i></li>
+            <li class="icons"><i class="fa-brands fa-twitter"></i></li> 
+            <li ><a href="#" class="design topElement" target="_blank" rel="noopener">English</a></li>
+            <li><a href="#" class="design topElement" target="_blank" rel="noopener">My Page</a></li>
+            <li ><a href="#" class="design topElement" target="_blank" rel="noopener">Logout</a></li>
+         </ul>
+      </div>
+        <nav class="navbar">
+        <a href="#" id="logo"><img src="images/logoheader.png" alt="Logopicture"></a>
+        <ul class="DesktopNav design">
+            <li><a href="about.html" class="design" rel="noopener">About</a></li>
+            <li><a href="#" class="design" target="_blank" rel="noopener">Program</a></li>
+            <li><a href="#" class="design" target="_blank" rel="noopener">join</a></li>
+            <li><a href="#" class="design" target="_blank" rel="noopener">Sponsor</a></li>
+            <li><a href="#" class="design" target="_blank" rel="noopener">News</a></li>
+         </ul>
+         <button class="navButton">E-Ticket</button>
+        <ul class="nav-menu design">
+            <li class="nav-item"><a href="about.html" class="design" rel="noopener">About</a></li>
+            <li class="nav-item"><a href="#" class="design" target="_blank" rel="noopener">Program</a></li>
+            <li class="nav-item"><a href="#" class="design" target="_blank" rel="noopener">join</a></li>
+            <li class="nav-item"><a href="#" class="design" target="_blank" rel="noopener">Sponsor</a></li>
+            <li class="nav-item"><a href="#" class="design" target="_blank" rel="noopener">News</a></li>
+         </ul>
+         <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </div>
+      </nav>
+  </header>
+                         <!--First Section-->
+  <section class="headlineSection">
+    <div class="content">
+    <p class="transparentText">"Let The Music Speak !"</p>
+    <h1 class="title">The music festival<br>Hangang Summer <br> 2022 </h1>
+    <div class="description">
+     <p>2022 marks the seventh year of the Hangang Summer Festival and we have planned a wide selection of over 80 programs that are designed to allow you to be a part of this grand picnic while enjoying 
+       the beautiful natural surroundings of Seoul’s landmark Hangang River.</p> 
+    </div>
+    <p class="date">2022 juin 25 (Sat)-11 july (Mon)</p>
+    <p class="place">@Bop Club and Bar and More</p>
+  </div>
+  </section>
+
+                            <!--Program section-->
+  <section class="program">
+  <h2 class="titre">Main program</h2>
+  <hr>
```

Comment:
- [ ] You did a great job on the styles but for this project, you are supposed to stick to the [Behance ](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) design as much as possible. When we hover the mouse over cards a white border around cards should appear as on the design, Please apply hover effects to your cards so the cards show a white border when we hover the mouse over them.

### See the image for reference 

![image](https://user-images.githubusercontent.com/83803414/157689981-91c089cf-d233-49a4-b0d5-3e74f7a6d17b.png)